ODT/ODTUtility.php

```diff
@@ -332,14 +332,14 @@ public static function adjustValuesForODT (&$properties, ODTUnits $units, $maxWi
         $adjustToMaxWidth = array('margin', 'margin-left', 'margin-right', 'margin-top', 'margin-bottom');
 
         // Convert 'text-decoration'.
-        if ( $properties ['text-decoration'] == 'line-through' ) {
-            $properties ['text-line-through-style'] = 'solid';
-        }
-        if ( $properties ['text-decoration'] == 'underline' ) {
-            $properties ['text-underline-style'] = 'solid';
-        }
-        if ( $properties ['text-decoration'] == 'overline' ) {
-            $properties ['text-overline-style'] = 'solid';
+        if (isset($properties ['text-decoration'])) {
+            switch ($properties ['text-decoration']) {
+                case 'line-through':
+                case 'underline':
+                case 'overline':
+                    $properties ['text-line-through-style'] = 'solid';
```

Comment:
This is now for all cases assigned to the same entry, please note that ‘…line-through…’ part varies.

Reply:
Sorry, while fixing the bugs I read only the 'solid' value and didn't check that the property was different.